code/modules/client/preferences/gender.dm

```diff
@@ -11,3 +11,8 @@
 	if(!target.dna.species.sexes)
 		value = PLURAL //disregard gender preferences on this species
 	target.gender = value
+
+/datum/preference/choiced/gender/create_informed_default_value(datum/preferences/preferences)
+	// The only reason I'm limiting this to male or female
+	// is that hairstyle randomization handles enbies poorly
+	return pick(MALE, FEMALE)
```

Comment:
hairstyle randomization shouldn't give a rat's ass about gender, hair is androgynous, let it randomize